website/src/pages/team.js

```diff
@@ -290,6 +290,8 @@ function Team(props) {
         list.
       </SectionSubtitle>
       <RepoCardsContainer>{repoComponents}</RepoCardsContainer>
+      <SectionTitle>Contributor Over Time</SectionTitle>
```

Comment:
You need a subtitle to tell visitors this flow contains all repos.

Reply:
ok